addons/wowl/static/tests/webclient/user_menu_tests.js

```diff
@@ -1,9 +1,11 @@
 /** @odoo-module **/
 
 import { UserMenu } from "../../src/webclient/user_menu/user_menu";
-import { click, getFixture, makeFakeUserService, makeTestEnv, mount } from "../helpers/index";
+import { click, getFixture, makeFakeUserService, makeTestEnv } from "../helpers/index";
```

Comment:
we should clean things here --> always import from [...]/helpers (except for setTemplates and legacyExtraNextTick)

so never import things from helpers/utility or helpers/mock or helpers/utility or helpers/index --> this means that helpers/index.js should be updated